Dockerfile

```diff
@@ -1,13 +1,19 @@
-FROM debian:stretch
+FROM debian:testing
```

Comment:
Why testing? Stable feels more... stable :)

Reply:
Doesn't work, ruby too old.

Reply:
Then let's base the image on Ubuntul LTS.

Reply:
bump?